### ISSUE TITLE
ABR: NullPointerException: No owning WizardImplementation for id _#UndeterminedStep

### ISSUE THREAD

**Issue by uckelman** [vassalengine/vassal#1842]
| Bugzilla |  |
|----------|----------|
| ID | 11330 |
| Reported | 2014-08-16 19:17:00 +0200 |
| Modified | 2014-08-17 15:29:05 +0200 |
| Product | VASSAL |
| Component | Unknown |
| Version | 3.2.13 |
| Platform | All |
| OS | All |
| Status | RESOLVED |
| Resolution | DUPLICATE |
| Priority | unspecified |
| Severity | normal |
| Milestone | --- |
| Reporter | [Joel Uckelman](mailto:uckelman@nomic.net) |
| Assignee | [Joel Uckelman](mailto:uckelman@nomic.net) |
| Duplicate of | 2558 |

| Attachments | |
|-------|-------|
| [errorLog](http://www.vassalengine.org/files/bz/10460) | [Joel Uckelman](mailto:uckelman@nomic.net) 2014-08-16 19:17:00 +0200 text/plain 20101 the errorLog |

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2014-08-16 19:17:00 +0200
<pre>chosing the map before starting</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2014-08-16 19:17:02 +0200
<pre>Created attachment 10460
the errorLog</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2014-08-17 15:28:17 +0200
<pre>*** This bug has been marked as a duplicate of bug 2258 ***</pre>

#### [Joel Uckelman](mailto:uckelman@nomic.net) 2014-08-17 15:29:05 +0200
<pre>*** This bug has been marked as a duplicate of bug 2558 ***</pre>
